sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -107,7 +107,56 @@ APIs for managing configurations for devices and modules.
 <details>
 
 ```csharp
+/// <summary>
+/// Gets a configuration on the IoT Hub for automatic device/module management
+/// </summary>
+/// <param name="configurationId">The unique identifier of the configuration.</param>
+/// <param name="cancellationToken">The cancellation token.</param>
+/// <returns>The retrieved automatic device/module management twin configuration</returns>
+public virtual async Task<Response<TwinConfiguration>> GetConfigurationAsync(string configurationId, CancellationToken cancellationToken = default)
+
+/// <summary>
+/// Create or update a configuration on the IoT Hub for automatic device/module management
```

Comment:
"Create or update an automatic device/module management twin configuration" to follow the restful pattern

Reply:
Also, similar to the create or update for device/module, add a note here about what the precondition value should be when creating and what it can be when updating